src/components/sticky-object.js

```diff
@@ -51,22 +51,22 @@ AFRAME.registerComponent("sticky-object", {
   _onRelease() {
     // Happens if the object is still being held by another hand
     if (this.el.is("grabbed")) return;
-
     if (
       this.data.autoLockOnRelease &&
       (this.data.autoLockSpeedLimit === 0 ||
-        this.el.body.velocity.lengthSquared() < this.data.autoLockSpeedLimit * this.data.autoLockSpeedLimit)
+        this.el.components["ammo-body"].getVelocity() < this.data.autoLockSpeedLimit)
```

Comment:
Perhaps `getVelocity` could be called `getSpeed` to indicate that it returns a scalar value